test/contrib/beam_dataflow_test.py

```diff
@@ -0,0 +1,314 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import luigi
+from luigi.contrib import beam_dataflow
+from luigi import local_target
+from mock import MagicMock, patch
+import unittest
+
+
+class TestDataflowParams(beam_dataflow.DataflowParams):
+    @property
+    def runner(self):
+        return "runner"
+
+    @property
+    def project(self):
+        return "project"
+
+    @property
+    def zone(self):
+        return "zone"
+
+    @property
+    def region(self):
+        return "region"
+
+    @property
+    def staging_location(self):
+        return "stagingLocation"
+
+    @property
+    def temp_location(self):
+        return "tempLocation"
+
+    @property
+    def gcp_temp_location(self):
+        return "gcpTempLocation"
+
+    @property
+    def num_workers(self):
+        return "numWorkers"
+
+    @property
+    def autoscaling_algorithm(self):
+        return "autoscalingAlgorithm"
+
+    @property
+    def max_num_workers(self):
+        return "maxNumWorkers"
+
+    @property
+    def disk_size_gb(self):
+        return "diskSizeGb"
+
+    @property
+    def worker_machine_type(self):
+        return "workerMachineType"
+
+    @property
+    def worker_disk_type(self):
+        return "workerDiskType"
+
+    @property
+    def job_name(self):
+        return "jobName"
+
+    @property
+    def service_account(self):
+        return "serviceAccount"
+
+    @property
+    def network(self):
+        return "network"
+
+    @property
+    def subnetwork(self):
+        return "subnetwork"
+
+    @property
+    def labels(self):
+        return "labels"
```

Comment:
Is it possible to just do `labels = "labels"`?`

Reply:
yes! thanks for the suggestion 👍